src/Controller/EnMarche/MyTeam/SenatorialCandidateMyTeamController.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
-namespace App\Controller\EnMarche\MyTeam;
-
-use App\Entity\Adherent;
-use App\Repository\CommitteeRepository;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
-use Symfony\Component\Routing\Annotation\Route;
-
-/**
- * @Route(path="/espace-senatoriales/mon-equipe", name="app_senatorial_candidate_my_team_", methods={"GET"})
- *
- * @Security("is_granted('ROLE_SENATORIAL_CANDIDATE')")
- */
-class SenatorialCandidateMyTeamController extends AbstractMyTeamController
-{
-    protected function getSpaceType(): string
-    {
-        return 'senatorial_candidate';
```

Comment:
do we need to remove `_base_senatorial_candidate_space.html.twig` too?

Reply:
yes indeed, fixed